packages/cli/src/Init.ts

```diff
@@ -428,6 +402,44 @@ export class Init implements Command {
       console.log(successMessage('Prisma Postgres® is ready to be used! 🚀'))
     }
 
+    if (
+      (isPpgCommand && (fs.existsSync(path.join(outputDir, 'schema.prisma')) || fs.existsSync(prismaFolder))) ||
+      fs.existsSync(path.join(prismaFolder, 'schema.prisma'))
```

Comment:
Do we want to show the message if there's an empty `prisma` folder but no schema file?

Reply:
We will not proceed with the schema.prisma or .env creation if:
- prisma folder exists
- prisma/schema.prisma exists
- ./schema.prisma exists

If any of these exists, we print success output when --db flag is present.

Reply:
I have updated the copy to include mentioning of the folder.